charts/ssi-credential-issuer/values.yaml

```diff
@@ -0,0 +1,317 @@
+
+dbConnection:
+  schema: "issuer"
+  sslMode: "Disable"
+
+postgresql:
+  # -- PostgreSQL chart configuration;
+  # default configurations:
+  # host: "issuer-postgresql-primary",
+  # port: 5432;
+  # Switch to enable or disable the PostgreSQL helm chart.
+  enabled: true
+  # -- Setting image tag to major to get latest minor updates
+  image:
+    tag: "15-debian-12"
+  commonLabels:
+    app.kubernetes.io/version: "15"
+  auth:
+    # -- Non-root username.
+    username: issuer
+    # -- Database name.
+    database: issuer
+    # -- Secret containing the passwords for root usernames postgres and non-root username issuer.
+    # Should not be changed without changing the "issuer-postgresSecretName" template as well.
+    existingSecret: "{{ .Release.Name }}-issuer-postgres"
+    # -- Password for the root username 'postgres'. Secret-key 'postgres-password'.
+    postgrespassword: ""
+    # -- Password for the non-root username 'issuer'. Secret-key 'password'.
+    password: ""
+    # -- Password for the non-root username 'repl_user'. Secret-key 'replication-password'.
+    replicationPassword: ""
+  architecture: replication
+  audit:
+    pgAuditLog: "write, ddl"
+    logLinePrefix: "%m %u %d "
+  primary:
+    # -- Extended PostgreSQL Primary configuration (increase of max_connections recommended - default is 100)
+    extendedConfiguration: ""
+    initdb:
+      scriptsConfigMap: "{{ .Release.Name }}-issuer-cm-postgres"
+  readReplicas:
+  # -- Extended PostgreSQL read only replicas configuration (increase of max_connections recommended - default is 100)
+    extendedConfiguration: ""
+
+externalDatabase:
+  # -- External PostgreSQL configuration
+  # IMPORTANT: non-root db user needs to be created beforehand on external database.
+  # And the init script (02-init-db.sql) available in templates/configmap-postgres-init.yaml
+  # needs to be executed beforehand.
+  # Database host ('-primary' is added as postfix).
+  host: "issuer-postgres-ext"
+  # -- Database port number.
+  port: 5432
+  # -- Non-root username for issuer.
+  username: "issuer"
+  # -- Database name.
+  database: "issuer"
+  # -- Password for the non-root username (default 'issuer'). Secret-key 'password'.
+  password: ""
+  # -- Secret containing the password non-root username, (default 'issuer').
+  existingSecret: "issuer-external-db"
+  # -- Name of an existing secret key containing the database credentials.
+  existingSecretPasswordKey: "password"
+
+# -- Provide details about centralidp (CX IAM) Keycloak instance.
+centralidp:
+  # -- Provide centralidp base address (CX IAM), without trailing '/auth'.
+  address: "https://centralidp.example.org"
+  authRealm: "CX-Central"
+  jwtBearerOptions:
+    requireHttpsMetadata: "true"
+    metadataPath: "/auth/realms/CX-Central/.well-known/openid-configuration"
+    tokenValidationParameters:
+      validIssuerPath: "/auth/realms/CX-Central"
+      validAudience: "Cl24-CX-Ssi-Credential-Issuer"
```

Comment:
Could you please make sure that the naming of the client is coherent with https://github.com/eclipse-tractusx/portal-iam/issues/66?

Reply:
adjusted

Reply:
Please also adjust it within the env values file

Reply:
forget those, adjusted